app/static/poll.js

```diff
@@ -82,4 +88,5 @@ function makeGraph(results) {
             }
         }
     });
-}
\ No newline at end of file
+}
+#ciao
```

Comment:
AHAH what is this 😄

Reply:
Not really a problem to leave comments, but always check not to leave **compromising** comments - I have seen stuff in some repositories that you can't imagine.